.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```

Comment:
⚠️ this will probably fail on window

Reply:
it works OK on windows

they stated in their FAQ:
Does it work on Windows?
Yes. When you install Git on Windows, it comes with the necessary software to run shell scripts.

Reply:
perfect